pkg/operator/config.go

```diff
@@ -43,6 +43,10 @@ const (
 	// DefaultClusterAutoscalerCloudProvider is the default name for
 	// the CloudProvider beeing used.
 	DefaultClusterAutoscalerCloudProvider = "openshift-machine-api"
+
+	// DefaultClusterAutoscalerVerbosity is the default logging
+	// verbosity level for ClusterAutoscaler deployments.
+	DefaultClusterAutoscalerVerbosity = 2
```

Comment:
I think the consensus on the channel was 1. so, LGTM.

Reply:
Done.